target/linux/rockchip/armv8/base-files/etc/board.d/02_network

```diff
@@ -35,6 +36,9 @@ rockchip_setup_macs()
 		wan_mac=$(nanopi_r2s_generate_mac)
 		lan_mac=$(macaddr_add "$wan_mac" +1)
 		;;
+	xunlong,orangepi-r1-plus)
+		wan_mac=$(macaddr_add "$(cat /sys/class/net/eth1/address)" -1)
```

Comment:
Can the base address be read from flash somehow? Having relative definitions for lan/wan is somewhat unpleasant when things change later ...

Reply:
Unfortunately no. If I understand correctly this is how the [vendor firmware](https://github.com/orangepi-xunlong/orangepi-build/blob/004741d87c3ad6022849a553b6c75aa2c199c2f5/external/packages/blobs/rockchip/set_r1plus_lan_mac.sh) sets the MAC address.
There are references to EEPROM with MAC address in the schematic, but it also says this part is NC (no connect).